open_academy/wizard/open_academy_wizard.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models, fields
+
+
+class OpenAcademyWizard(models.TransientModel):
+    _name = "open.academy.wizard"
+    _description = "wizard model on open academy"
+
+    def _default_sessions(self):
```

Comment:
This method should be after fields see guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models
```suggestion
    def _default_session(self):
```

When you move it you need to change the field and use lambda function in the default parameter to find the method.
`default=lambda self: self._default_session()`